state/logs.go

```diff
@@ -27,7 +27,7 @@ type GrantLogsEncryptionKey struct {
 	Plaintext      string `json:"plaintext"`
 }
 
-const LogStreamBufferLen = 500
+const LogStreamBufferLen = 10
```

Comment:
I'm not sure why we use such a large buffer here (won't the channel immediately be read from?), but #606 apparently fixes the real issue so I guess I'll drop this change and make this a regex-only PR.